.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: scala
+scala:
+- 2.11.8
+jdk:
+- oraclejdk8
+script:
+- sbt test
+before_deploy:
+- pip install --user release-manager==0.2.0
```

Comment:
I followed this style: https://github.com/snowplow/snowplow/blob/master/.travis.yml#L1-L13

But I can change it to your suggestion if you think it's better